addons/event_crm/tests/common.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import datetime
+
+from dateutil.relativedelta import relativedelta
+from odoo.addons.event.tests.common import TestEventCommon
+
+class TestEventCommon(TestEventCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestEventCommon, cls).setUpClass()
```

Comment:
do you really need TestEventCommon setup ?
If no you can juste inherit from savepointcase.
if yes, watch out you don't create data that interfere with already existing tests.

Reply:
Up :arrow_up:  it seems you create your own data, in which case you could maybe just inherit from savepointcase. I let you check :) .